src/components/molecules/Monaco/editorConstants.ts

```diff
@@ -1,15 +1,21 @@
 import {monaco} from 'react-monaco-editor';
 
+import Colors from '@styles/Colors';
+
+export const MODEL_OWNER = 'monokle';
```

Comment:
TODO: see whether having your own model identifier clashes with the yaml key. If you use `yaml` it does not work properly. I expect because the `monaco-yaml` package clears markers that we set as Monokle.

Reply:
good enough. It seems to not interfere but will need to test more thoroughly later.